inline_verifier.go

```diff
@@ -0,0 +1,223 @@
+package ghostferry
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/golang/snappy"
+	"github.com/sirupsen/logrus"
+)
+
+type InlineVerifier struct {
+	SourceDB *sql.DB
+	TargetDB *sql.DB
+
+	sourceStmtCache *StmtCache
+	targetStmtCache *StmtCache
+	logger          *logrus.Entry
+}
+
+func (v *InlineVerifier) VerifyBeforeCutover() error {
+	// TODO: Iterate until the reverify queue is small enough
```

Comment:
This method will be implemented when the binlog portion of the InlineVerifier is hooked up